src/Xamarin.Android.Build.Tasks/Tasks/GetAotArguments.cs

```diff
@@ -51,13 +51,24 @@ public abstract class GetAotArguments : AndroidAsyncTask
 
 		public ITaskItem [] Profiles { get; set; } = Array.Empty<ITaskItem> ();
 
-		public bool UsingAndroidNETSdk { get; set; }
-
-		public string AotAdditionalArguments { get; set; } = "";
-
 		[Required, Output]
 		public ITaskItem [] ResolvedAssemblies { get; set; } = Array.Empty<ITaskItem> ();
 
+		[Output]
+		public string? Triple { get; set; }
```

Comment:
Naming suggestion: `TargetTriplet` (as per [autoconf docs](https://www.gnu.org/software/autoconf/manual/autoconf-2.65/html_node/Specifying-Target-Triplets.html)) instead of `Triple`.  `Triple` is not necessarily "fully semantically meaningful."

Reply:
Neither `TargetTriplet` or `Triple` really mean anything to me -- so open to suggestions.

The `<MonoAOTCompiler/>` task uses `Triple`, which passes `mtriple` to the actual AOT compiler:

https://github.com/dotnet/runtime/blob/bdc6ad73834a13e80f131e402e883903c31ea172/src/tasks/AotCompilerTask/MonoAOTCompiler.cs#L573-L576

We could use a different name, but is that worse? We'd still be using `Triple` on the next task unless we also renamed it -- and also got that change backported.